03_cross_cancer_classification/plot_flip_labels_results.ipynb

```diff
@@ -0,0 +1,559 @@
APw7owrFeY0+NrTh0DvXhiVgI+nvY2V4mpubz/0wFawYpRIYQQQogGteseGEII4Wxtaa11a8vN/0pcRGbxUXSKjvmjn8bX6MNr/0tmZ1ouAPFdA3l8dgL6NlK8zsgu5qVPdzmKM5OGdmHW+O6ObXeFEHZtKS8LIURb1+hd2sGDB9m1axe7d+/m1ltvZcuWLc6ISwghRAMkN5+/gWdmYaiayp7T+1ix9YijeBHk58F90/q2meIFQHS4P4/M6I9Bb39Nq7Yf47vNma4NSgghhBDiIjR6p/aPf/wDk8nE4sWLefTRR1m0aJEz4hJCCNEAyc3nLyG0ehnJxiM7+WpDOgAGvcKD1/TD38fkqtBaTHxUEA9c0xe9zj7r4uuNGaxOPObiqIQQQgghLkyjBQyDwUCPHj2wWCwkJCRgs9mcEZcQQogGSG4+fx28gunqFwnAsbJMNF0VADdN6klMhL8rQ2tRCd07/H/27js8jupc/Ph3ynb1alm2ZFu23CR3cLfpAScQwAFTAgmQAkluGtwbkhuIkziUm/JLo4RcCFxTTCB0CN1gjMHgIktykWzJlmRZvW/fnZnfHyutLEsrybaapfN5Hj/anZmdOSOvj2feec97uOVLM+kYOPLMuwf5uKBqWNskCIIgCIJwKvoMYEiSxO23386qVat44403sNlsQ9EuQRAEoReibz41cxJzQi8kAyW+llVzx7Nq7vjhbdQQWDJrHF/9wvTw+8fe2M/OotphbJEgCIIgCMLJ67OIZ2NjIwUFBaxatYrt27czY8YM4uLihqh5IWdaoThBEIRIBqpYnOibT55hGDz4xnb22V4AwOJJ4/4v/ACTOnrqXvTljU/LeP6DEiA0dOYHX5nL7MkJw9wqQRheooinIAjCmaPPq7ZvfOMbHD16FJfLxZIlS4b8AlkQBEHoTvTNJ2/z7kp2FrjRXaGblaC9loDhG+ZWDa01SzJZsyQTgKBm8JcX8jlU2TLMrRIEQRAEQeifPgMYjzzyCF6vl6997Wvceeed7Ny5cyjaJQiCIPRC9M0n59DRFp559yAAelMaAJqhUVC/bzibNSzWrp7COfPTAfAHdP74zz2U17QNc6sEQRAEQRD61mcAIykpiVtuuYW//OUv+Hw+brvttqFolyAIgtAL0Tf3X7PTxwMvFaDpoRGTF2WfFV63uy5/uJo1bCRJ4qsXZbNkVioAbl+QPzybR03jmTMcSBAEQRCEsanPAMZLL73E1772NX7yk5+wevVqtmzZMhTtEgRBEHoh+ub+CWo6D75USIszNOPI2TNTuHLxHMY7xgGwv6EYT9AznE0cFrIkcfMXZzI3KxGAVneA323aTWOrd5hbJgiCIAiCEJna1wYHDhzg7rvvJisrayjaIwiCIPSD6Jv759n3D3HoaKjGQ3qyg5sumYkkSSxImcOxw9UEDY2C+v2cPW7BMLd06KmKzG2X5/DH5/ZwoLyZhlYfv9uUx53XLyDGYR7u5gmCIAiCIHTT5ywkI8GZVuleEAQhktFU7X6k983bCqv439f2A2CzqNz9tUWkJtgBqHbV8OvtvwdgbtJsvjXna8PWzuHm8QX53abdHK4K1cHISIniv66bj91qGuaWCcLQGE39siAIwmg3duaOEwRBEMaMsuo2nnizKPz+m5fOCgcvAMY5UhnnCNWA2NtYhDc4dodO2CwqP7p6HuOTHACU1zr54/P5+ALaMLdMEARBEAShq5MOYHi9Y/ciTxAEYaQSfXMnpyfAAy8WEAjqAFy2fBLzpiZ1225Bci4AQT1IYcOBIW3jSBNlM3H7unkkxVqB0KwtD7xQQFDTh7llgiAIgiAInSIGMCorK9mwYQN//vOf8XhCBc4+/PBDLr300iFrnCAIgtCV6Jt7p+sGj7yyl/qWUEBnTlYil62Y3OO281PmhF/vri0YkvaNZPHRFu64dj6xUaH6F4WHG3nklb3o+ogfaSoIgiAIwhgRsQbGNddcwxVXXMGxY8fw+/2YTCbefvttNmzYwKJFi4a0kSN9nLUgCEJ/ne5Ya9E39+6FLSW8tq0MgJQ4G3d9fRGOCLUcDMPg19t/R427DpNs4v6Vv8CiiOKVlXVO7n96N05PAIAVc9K46ZIZSJI0zC0ThMEhamAIgiCcOSLOQiJJEuvWrQPgvPPO46yzzuLll1/GYrEMWeMEQRCErkTfHNmu4rpw8MJskvnelbkRgxcQ+l3OT87lzbL3CegB9jYcYMFxWRljVXpyFD+6ei6/fWY3Xr/G1vwq7BaVdedNFUEMQRAEQRCGVcQhJKraGduIi4vjvvvuExfIgiAIw+xU+2Zd17n77rtZt24dN9xwA2VlZV3Wv/XWW6xdu5avfOUrPPfccwPe7sFW1eDif1/bF35/0yUzmZAS1efnug4jyR+Utp2JJqfF8IOvzMGkhi4T3v68gle3HRneRgkRabrOR3uOcc/Gndzx4Mfcs3EnH+05Job/CIIgCKNOrxkYHaKiosRTF0EQhBHgVPvmd999F7/fz7PPPkteXh733XcfDz30EACapvH73/+ef/3rX9jtdtasWcP5559PQkLCoJzDQPP4gvz1hQK8/tCsGRedNZHFs1L79dn0qDSSbYnUeRoobDiAX/NjFsNIAJieEc93Ls/hry8UoOkGL310GJtF5cJFE4e7acJxNF3n4Zf2srO4LryssdXHocoW8ksauPXy2SiymHROEARBGB0iBjB27drFihUrAGhubg6/Bti6devgt0wQBEHo5lT75p07d7Jy5UoA5s2bR2FhYXidoii88cYbqKpKQ0MDAA6Ho8+2KIpEXJy9z+0Gk2EYPPLUTqoaQrU4cqYk8s3Lc1GU/t+wLZm4gFeL38Gv+SnzHuGs9HmD1Nozz+pFGciqwh+f3Y1hwDPvHqS22cPROhcNzR4S42xcsGgi5yyciCKLBx3D4b3Py7sEL463s7iOvNImzhdBJ0EQBGGUiBjAOP7i9mTpus769espKirCbDazYcMGMjMzw+tfe+
```

Comment:
Idk how to do it but you could probably put the color key in its own panel in the bottom right

Reply:
Yeah, I think if this were a figure for a publication or something like that I'd try to put a color key there, but these experiments are mostly just a sanity check so I don't want to spend too much time on it now (and my gut feeling is that it would take a decent amount of tweaking/trying things to get it to work right).

If I end up using this code (or similar code) in future analyses, I'll go back and fix the key/legend.